Content.Tests/DMProject/Tests/Expression/ternary_2.dm

```diff
@@ -0,0 +1,13 @@
+
+//# issue 666
+
+/obj/o
+	var/test = 5
+
+/proc/get_obj()
+	return (new /obj/o)
+
+/proc/RunTest()
+	var/obj/o/thing
+	var/c = (thing = get_obj()).test
+	ASSERT(c == 5)
```

Comment:
The name of this test is `ternary_2` but there are no uses of ternaries in here.